packages/console/src/pages/SignInExperience/tabs/SignUpAndSignIn/components/SignInMethodEditBox/utilities.ts

```diff
@@ -12,6 +15,28 @@ export const getSignInMethodPasswordCheckState = (
   return isSignUpPasswordRequired || defaultCheckState;
 };
 
-export const getSignInMethodVerificationCodeCheckState = (signInIdentifier: SignInIdentifier) => {
-  return signInIdentifier !== SignInIdentifier.Username;
+export const getSignInMethodVerificationCodeCheckState = (
+  signInIdentifier: SignInIdentifier,
+  signUpConfig: SignUpForm,
+  currentCheckState?: boolean
+) => {
+  if (signInIdentifier === SignInIdentifier.Username) {
+    return false;
+  }
```

Comment:
should return the currentCheckState SSOT?

Reply:
No, If the sign-in method is Username, the verification code state is set to false by default.